webcompat/templates/contributors/alumni.html

```diff
@@ -0,0 +1,305 @@
+    <img src="https://avatars0.githubusercontent.com/u/562354" alt="Ola Gasidlo" class="bio-img">
+  </div>
+  <div class="grid-nested x1">
+    <div class="grid-cell bio-desc x1">
+      <h4 class="headline-2 heading">Ola Gasidlo</h4>
+      <h4 class="headline-3">Germany</h4>
+      <p>
+        Loves making the web accessible for everyone, break things and daughter-driven development.
+      </p>
+    </div>
+    <p class="grid-cell bio-links x1">
+      <a class="bio-icon-twitter" href="http://twitter.com/misprintedtype" title="misprintedtype on twitter" target="_blank">twitter@misprintedtype </a> <br>
+      <a class="bio-icon-github" href="http://github.com/zoepage" title="zoepage on github" target="_blank">github@zoepage </a>
+    </p>
+  </div>
+</article>
+
+<!-- Trishul -->
+<article class="grid-row bio">
+    <div class="grid-cell x1-fix">
+      <img src="https://avatars3.githubusercontent.com/u/5318732" alt="Trishul" class="bio-img">
+    </div>
+    <div class="grid-nested x1">
+      <div class="grid-cell bio-desc x1">
+        <h4 class="headline-2 heading">Trishul</h4>
+        <h4 class="headline-3">Germany</h4>
+        <p>#mozillian #frontEndDev #reactDev #webExtensions #teamWeb</p>
+      </div>
+      <p class="grid-cell bio-links x1">
+        <a class="bio-icon-twitter" href="https://twitter.com/trishulgoel" title="trishulgoel on twitter" target="_blank">twitter@trishulgoel </a> <br>
+        <a class="bio-icon-github" href="https://github.com/tsl143" title="tsl143 on github" target="_blank">github@tsl143 </a>
+      </p>
+    </div>
+</article>
+
+<!-- Swarnava Sengupta -->
+<article class="grid-row bio">
+  <div class="grid-cell x1-fix">
+    <img src="https://avatars0.githubusercontent.com/u/1162991" alt="Swarnava Sengupta" class="bio-img">
+  </div>
+  <div class="grid-nested x1">
+    <div class="grid-cell bio-desc x1">
+      <h4 class="headline-2 heading">Swarnava Sengupta</h4>
+      <h4 class="headline-3">India</h4>
+      <p>
+        Mozilla QA and User Support contributor, leading the featured add-ons board for addons.mozilla.org.
+      </p>
+    </div>
+    <p class="grid-cell bio-links x1">
+      <a class="bio-icon-twitter" href="https://twitter.com/sw4rn4v4" title="sw4rn4v4 on twitter" target="_blank">twitter@swarnava </a> <br>
+      <a class="bio-icon-github" href="https://github.com/swarnava" title="swarnava on github" target="_blank">github@swarnava </a>
+    </p>
+  </div>
+</article>
+
+
+<div class="grid-row">
+        <div class="grid-cell x3">
+          <a href="https://github.com/orgs/webcompat/people" title="You can find more contributors on Github" target="_blank">and many more...</a>
+        </div>
+      </div>
+  </section>
+      </div>
+    </section>
+  </main>
+</div>
+{% endblock %}
+{%- block extrascripts -%}
+{%- if config.PRODUCTION or config.STAGING -%}
+<script src="{{ url_for('static', filename='js/dist/contributors.min.js')|bust_cache }}"></script>
```

Comment:
I think @ksy36 will want to rebase after I merge this, to ensure the script references are correct.

SORRY KSENIA